gossip/gossip.go

```diff
@@ -274,7 +300,7 @@ func (g *Gossip) GetInfosAsJSON() ([]byte, error) {
 // of info denoted by key. The contentsChanged bool indicates whether
 // the info contents were updated. False indicates the info timestamp
 // was refreshed, but its contents remained unchanged.
-type Callback func(key string, contentsChanged bool, content interface{})
+type Callback func(key string, contentsChanged bool, content []byte)
```

Comment:
perhaps content should be changed to `proto gogoproto.Message`?
I don't see why you would ever want the raw bytes if we're enforcing gossiping only protos.